homeworks/01-bubble-sort/src/components/Helpers.tsx

```diff
@@ -0,0 +1,11 @@
+export const randomNum = (max: number = 100, min: number = 1): number => (

```

Comment:
This is not a component. `Helpers.tsx` is not suitable here (e.g., `utilts.ts` much better). And it should be located separately, not inside the `components` folder.